lib/rucio/core/replica_sorter.py

```diff
@@ -158,6 +158,59 @@ def __get_distance(se1, client_location, ignore_error):
     return cache_val
 
 
+custom_distance_table = {}
+custom_distance_download_time = None
+
+
+def __download_custom_distance_table() -> None:
```

Comment:
Please add the usual method description etc. I know this is an internal method, but it would be helpful to also explain the format of the downloaded table, possible values, etc.

Reply:
I have added the method description including the table format.